packages/manager/modules/adp/src/adp.module.js

```diff
@@ -0,0 +1,31 @@
+import angular from 'angular';
+import 'ovh-ui-kit/dist/oui.css';
+
+import './adp.scss';
+import adpComponent from './adp.component';
+import adpService from './adp.service';
+import adpConstants from './adp.constants';
+import deploy from './deploy';
+import service from './service';
+
+const moduleName = 'ovhManagerAdpComponent';
+
+angular
+  .module(moduleName, [deploy, service])
+  .config(/* @ngInject */($stateProvider) => {
+    $stateProvider.state('adp', {
+      url: '/adp',
+      component: 'adpComponent',
+      translations: {
+        value: [
+          '.',
+        ],
+        format: 'json',
+      },
+    });
+  })
+  .component('adpComponent', adpComponent)
+  .service('adpService', adpService)
+  .constant('adpConstants', adpConstants);
```

Comment:
Prefer exposing each constant separately, or you can also import required constant with `import/export` : 

In your module file : 
```js
import { PUBLIC_CLOUD_STATUS } from './adp.constants';

angular.module('...', [...])
  .constant('ADP_PUBLIC_CLOUD_STATUS', PUBLIC_CLOUD_STATUS);
```

And use it by injecting the constant : 
```js
export default class MyController {
  /* @ngInject */
  constructor(ADP_PUBLIC_CLOUD_STATUS) {
    this.ADP_PUBLIC_CLOUD_STATUS = ADP_PUBLIC_CLOUD_STATUS;
  }
  
  getStatus(status) {
    return this.ADP_PUBLIC_CLOUD_STATUS[status]; // kis
  }
}
```

But, you could also import the constant directly  : 

```js
import { PUBLIC_CLOUD_STATUS } from '../../adp.constants';

export default class MyController {
  getStatus(status) {
    return ADP_PUBLIC_CLOUD_STATUS[status]; // kis
  }
}
```

Reply:
The constants are being imported and created individually now